### DISCUSSION TITLE
How do I handle updates for multiple operating systems?

### DISCUSSION THREAD

**Discussion by ZeroMax-dev** [dennisvang/tufup#158] [Q&A] [answered]
So I have built a Windows .exe with PyInstaller and added it as bundle to my repository.
Now I have `MyApp-0.1.tar.gz` in my targets.

If I create a build on Linux and do the same again it would not be added to the repository because tufup thinks I'm adding the same thing again and tells me that I need to increase the version number.

How can I provide multiple bundles for different operating systems (Windows, macOS, Linux) for the same `APP_VERSION`?
And how can I tell my app which bundle to download and update?

**Comment by dennisvang** [comment, marked as answer]
Hi @ZeroMax-dev thanks for the interesting question.

Unfortunately, tufup currently only supports a single platform per repository. 

Supporting multi-platform repositories is still on the to-do list, as discussed in #79.

**Comment by ZeroMax-dev** [comment]
Hey @dennisvang, thanks for your prompt answer! And thank you very much for tufup, it's been pretty good so far!

Using separate repositories would have been my way to go as well.

Thanks for your engagement!
